<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acldisplay</class>
 <widget class="QMainWindow" name="acldisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>acldisplay</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="pathWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>State</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>381</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="fileLabel">
        <property name="text">
         <string>File: </string>
        </property>
       </widget>
       <widget class="QLabel" name="fileNameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="ownerLabel">
        <property name="text">
         <string>Owner: </string>
        </property>
       </widget>
       <widget class="QLabel" name="ownerNameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="groupLabel">
        <property name="text">
         <string>Group: </string>
        </property>
       </widget>
       <widget class="QLabel" name="groupNameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
